docs/static/releasenotes.asciidoc

```diff
@@ -59,6 +59,14 @@ This section summarizes the changes in the following releases:
 [[logstash-7-16-1]]
 === Logstash 7.16.1 Release Notes
 
+[[known-issue-7-16-1]]
+==== Known issue
+
+Release 7.16.1 includes updated versions of filters that will not run on JDK 8.
+We are actively patching the build scripts for all filters that are affected. 
+
+Workaround: Clear the `JAVA_HOME` variable to use the JDK version (Java 11) bundled with Logstash. 
```

Comment:
```suggestion
Workaround: since Logstash bundles Java 11 (all distributions except noarch), clearing `JAVA_HOME` variable from your environment will enable Logstash to run with the bundled Java where this issue is not present.
```

Reply:
@yaauie - Do you think we should mention that they need to clear the `LS_JAVA_HOME` variable too? 

The documentation states the following in the [Bundled JDK](https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html#ls-jvm):

```Use the LS_JAVA_HOME environment variable if you want to use a JDK other than the version that is bundled.  ```

But later on in [LS_JAVA_HOME and JAVA_HOME](https://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html#java-home) we have:

```Logstash uses the Java version set in LS_JAVA_HOME. The LS_JAVA_HOME environment variable must be set for Logstash to operate correctly. If Logstash doesn’t find LS_JAVA_HOME it tries to fall back to JAVA_HOME. The usage of JAVA_HOME is now considered deprecated in favor of LS_JAVA_HOME.```

 
Side note: If Logstash will work with the default JDK if neither of these variables are set then we should clarify the above statements in our documentation.  If my understanding is correct, I will create a separate docs issue to address the above.  CC: @karenzone

Reply:
Thanks! Added LS_JAVA_HOME info.